internal/provider/vsphere/ignition.go

```diff
@@ -1,10 +1,41 @@
 package vsphere
 
-import "github.com/openshift/assisted-service/internal/common"
+import (
+	"os"
+	"path"
+	"path/filepath"
+
+	"github.com/openshift/assisted-service/internal/common"
+)
 
 func (p vsphereProvider) PreCreateManifestsHook(cluster *common.Cluster, envVars *[]string, workDir string) error {
 	return nil
 }
-func (p vsphereProvider) PostCreateManifestsHook(cluster *common.Cluster, envVars *[]string, workDir string) error {
+func (p vsphereProvider) PostCreateManifestsHook(_ *common.Cluster, _ *[]string, workDir string) error {
```

Comment:
for reference, here is UPI cI:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh#L189-L195

Reply:
Yeah I think this link belongs inside the code as a comment